run-frontend.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.8-slim
+
+COPY --from=neuralet/smart-social-distancing:latest-frontend /frontend/build /srv/frontend
+
+COPY ui/requirements.txt /ui/
+WORKDIR /ui
+
+RUN python3 -m pip install --upgrade pip setuptools==41.0.0 && pip install -r requirements.txt
+COPY ui/ /ui
+COPY libs/config_engine.py /ui/
+COPY config-frontend.ini /ui/
+
+#EXPOSE 8000
```

Comment:
is this meant to be removed?

Reply:
Yes, it's preferred to use no expose in the docker file as the port is configurable in the config file and in the instructions, we instruct to use the docker run with "-p" option